newIDE/app/src/InstancesEditor/index.js

```diff
@@ -1055,13 +1059,20 @@ export default class InstancesEditor extends Component<Props> {
   pauseSceneRendering = () => {
     if (this.nextFrame) cancelAnimationFrame(this.nextFrame);
     this._renderingPaused = true;
+    // Deactivate interactions when the scene is paused.
+    // Useful when the scene is paused to reload textures. The event system
+    // might try to check if pointer is over a PIXI object using the texture
+    // of the object. If there is no texture, it will crash.
+    // The PIXI.EventSystem is not based on the PIXI.Ticker.
+    this.instancesRenderer.getPixiContainer().eventMode = 'none';
```

Comment:
nice 👀 really nice, I did read about the new EventSystem & Ticker too.
This seems like this change is safe to me.